docs/development/workflow/virtualenv_detail.rst

```diff
@@ -1,105 +0,0 @@
-:orphan:
-
-.. _using-virtualenv:
```

Comment:
Removing this ref might break some docs downstream. Example report along similar vein: #11754

Reply:
Good point.  A quick search turned up at least one project that references that page: https://pypi.org/project/pywicta/0.1.dev19/

Good opportunity to try out sphinx-redirects.

Reply:
I added a redirect using sphinx-reredirects (I kept missing the "rere" in the name before).

@pllim Since this adds a new doc build dependency do you think I should make a separate PR for that first, or just go ahead and do it here since this is where the need arose?